internal/pkg/db/test/db_data.go

```diff
@@ -422,7 +424,7 @@ func testDBEvents(t *testing.T, db interfaces.DBClient) {
 		t.Fatalf("There should be 100 events, not %d", len(events))
 	}
 
-	events, err = db.EventsOlderThanAge(0)
+	events, err = db.EventsOlderThanAge(afterTime + 10)
```

Comment:
Magic constant `10`. Please define it as a `constant` at the top of the file and give it a meaningful, self-descriptive name.